src/installer/pkg/sfx/installers.proj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <ItemGroup>
+    <InstallerProjectReference Include="Microsoft.NETCore.App/*.sfxproj" />
+    <InstallerProjectReference Include="installers/*.proj" />
+    <InstallerProjectReference Include="bundle/*.bundleproj" />
+     
+  </ItemGroup>
+  <Target Name="BuildInstallerProjects" BeforeTargets="Build">
+    <MSBuild Projects="@(InstallerProjectReference)"
+             Targets="GenerateInstallers"
```

Comment:
Wouldn't the Microsoft.Build.Traversal SDK make more sense for this project? As I understand this is just a wrapper to build all installers, the shared framework and the bundles.

Reply:
The Traversal SDK here won't work because we can't set which target the traversal project's Build target executes (it's always the default target).

Reply:
Hmm I thought this should work: https://github.com/dotnet/arcade/blob/5537c88d462e239a42375e19cc0ec2ab1ebeec64/src/Microsoft.DotNet.GenFacades/build/Microsoft.DotNet.GenFacadesNotSupported.targets#L22.

Reply:
For more context, see the discussion in https://github.com/microsoft/MSBuildSdks/issues/214

Reply:
As discussed offline, this seems not to work in the traversal SDK.